vm/build.rs

```diff
@@ -1,10 +1,18 @@
-
 #[cfg(feature = "test")]
 mod build {
     extern crate lalrpop;
+
     pub fn main() {
-        lalrpop::Configuration::new().process_current_dir().unwrap();
-        println!("cargo:rerun-if-changed=src/core/grammar.lalrpop");
+        let out_dir = ::std::env::var("OUT_DIR").unwrap();
+        let manifest_dir = ::std::env::var("CARGO_MANIFEST_DIR").unwrap();
+
+        lalrpop::Configuration::new()
+            .set_out_dir(&out_dir[..])
+            .set_in_dir(&manifest_dir[..])
+            .process_file("src/core/grammar.lalrpop")
+            .unwrap();
```

Comment:
There is a [`use_cargo_dir_conventions`](https://github.com/nikomatsakis/lalrpop/blob/master/lalrpop/src/api/mod.rs#L64-L70) option... would this simplify things?

Reply:
Ah, that is a bit better. Was struggling to get the directories right so I just coded them all explicitly to make sure they were right.